src/stores/nodeDefStore.ts

```diff
@@ -137,6 +142,15 @@ export class ComfyNodeDefImpl
     this.nodeSource = getNodeSource(obj.python_module)
   }
 
+  /**
+   * Initializes API node fields
+   */
+  async initApiNodeFields() {
```

Comment:
I don't think we should send individual request to fetch api node cost. The cost should be fetched together in a single request.